dotfiles.sh

```diff
@@ -31,6 +31,7 @@ while [ $# -gt 0 ]; do
         -q|--quiet    ) quiet=yes;        shift ;;
         -f|--force    ) force=yes;        shift ;;
         -p|--preserve ) preserve=yes;     shift ;;
+        -e|--edit     ) edit=yes;         shift ;;
```

Comment:
You should pre-initialize the variable `edit` beforehand similar to what is done for the other options.

Reply:
Done